plugins/woocommerce/src/Caching/ObjectCache.php

```diff
@@ -111,7 +111,7 @@ public function get_default_expiration_value(): int {
 	 * @return CacheEngine
 	 */
 	private function get_cache_engine(): CacheEngine {
-		if ( null === $this->cache_engine ) {
+		if ( $this->cache_engine === null ) {
```

Comment:
sorry, you might wanna undo this, and similar this things in the file

Reply:
I know ☹